tests/Unit/Visualization/Python/Test_GenerateXdmf.py

```diff
@@ -6,12 +6,92 @@
 import spectre.Informer as spectre_informer
 import unittest
 import os
+from spectre.IO import H5 as spectre_h5
+import spectre.DataStructures as ds
 
 
 class TestGenerateXdmf(unittest.TestCase):
+    def setUp(self):
+        # We create a vol file using the python wrapper.
+        self.file_name = os.path.join(spectre_informer.unit_test_path(),
+                                      "Visualization/Python/VolTestData.h5")
```

Comment:
At the point this change will come in the commit order it will be, but this is just the same name as the file the that the test used before so it can be changed.

Reply:
Okay, it would be good to name it something include the name of the test. that way it will be unique.

Related: I remember we discussed at a call add a `unit_test_build_path` and ` unit_test_src_path` to avoid writing into the build directory. Where did we decide to add that and who was going to do it?